draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -121,14 +121,14 @@ Disclosure:
 
 
 Key Binding:
-:  Ability of the Holder to prove legitimate possession of an SD-JWT by proving
+:  Ability of the Holder to prove possession of an SD-JWT by proving
   control over a private key during the presentation. When utilizing Key Binding, an SD-JWT contains
   the public key corresponding to the private key controlled by the Holder (or a reference to this public key).
 
 Key Binding JWT (KB-JWT):
-:  A JWT for proving Key Binding as defined in (#kb-jwt).  A Key Binding JWT is
-   said to "be tied to" a particular SD-JWT when its payload includes a hash of the
-   SD-JWT in its `sd_hash` claim.
+:  A Key Binding JWT is said to "be tied to" a particular SD-JWT when its payload
+  includes a hash of the SD-JWT in its `sd_hash` claim.A JWT for proving Key Binding
```

Comment:
"One nit: on the Key Binding JWT def: You need a space between the sentences." 
thanks @debcooley 
```suggestion
  includes a hash of the SD-JWT in its `sd_hash` claim. A JWT for proving Key Binding
```

Reply:
perfect.  TY